experimental/handoff-library/csharp_dotnetcore/Microsoft.Bot.Builder.HandoffBot/publish.cmd

```diff
@@ -0,0 +1,3 @@
+nuget restore
+msbuild EchoBot.sln -p:DeployOnBuild=true -p:PublishProfile=arturl-handoff-bot-bd6d-Web-Deploy.pubxml -p:Password=0xscoKlgAzQZ38o6369ctvpkcPFBtNi49e6013ZduaRYgon6vwDMy9nwe5LY
+
```

Comment:
BUG BUG BUG - Why is there a password in here? 

Please Immediatly:
1. Invalidate this password.
2. Remove it from this file.